src/commands/push-image.yml

```diff
@@ -18,11 +18,18 @@ parameters:
     type: string
     default: "latest"
 
+  return-sha-var:
+    description: Variable you want set with the container sha (default = SHA256)
+    type: string
+    default: SHA256
+
 steps:
   - run:
       name: Push image to Amazon ECR
       command: |
         IFS="," read -ra DOCKER_TAGS \<<< "<< parameters.tag >>"
         for tag in "${DOCKER_TAGS[@]}"; do
-          docker push $<<parameters.account-url>>/<<parameters.repo>>:${tag}
+          PUSH=$(docker push $<<parameters.account-url>>/<<parameters.repo>>:${tag})
+          echo $PUSH
         done
+        export <<parameters.return-sha-var>>=$(echo $PUSH | grep sha256 | cut -d " " -f3)
```

Comment:
To access this env var from another command within the same job, this must be written to the BASH_ENV

Example: 

```
echo `export PATH="$PATH:<<parameters.install-path>>"` >> $BASH_ENV
```

Source: https://circleci.com/docs/2.0/orbs-best-practices/#parameters

Reply:
Gotcha thank you!